test/builders/statediffs.go

```diff
@@ -1,28 +1,40 @@
 package builders
 
 import (
+	"github.com/orbs-network/orbs-spec/types/go/primitives"
 	"github.com/orbs-network/orbs-spec/types/go/protocol"
 )
 
 type sdiff struct {
 	builder *protocol.ContractStateDiffBuilder
+	records []*protocol.StateRecordBuilder
 }
 
 func ContractStateDiff() *sdiff {
 	return &sdiff{
 		builder: &protocol.ContractStateDiffBuilder{
 			ContractName: "BenchmarkToken",
-			StateDiffs: []*protocol.StateRecordBuilder{
```

Comment:
See my comment above for the CommitStateDiff builder. 
I think this should not be removed. By default, ContractStateDiff should be built with values.